src/modules/AutoInstallRules.rb

```diff
@@ -61,7 +63,7 @@ def reset
       @installed_product = ""
       @installed_product_version = ""
       @hostname = ""
-      @hostaddress = ""
+      @hostaddress = nil
```

Comment:
NP: Actually uninitalized instance `@variables` are set to `nil` by default, we could remove this line completely...

But we may keep it to be consistent with the other values.